datasets/fusiongallery.py

```diff
@@ -61,7 +61,7 @@ def load_one_graph(self, file_path):
         sample = super().load_one_graph(file_path)
         # Additionally load the label and store it as node data
         label = np.loadtxt(
-            self.path.joinpath("breps").joinpath(file_path.stem + ".seg"), dtype=np.int, ndmin=1
+            self.path.joinpath("breps/seg").joinpath(file_path.stem + ".seg"), dtype=np.int, ndmin=1
```

Comment:
Can we have an argument in the dataloader that takes in the dataset version? Directly changing the path will break compatibility with s1.0.0.

Reply:
Good point.   Maybe we can work this out on the fly